app/controllers/projects/samples_controller.rb

```diff
@@ -8,12 +8,15 @@ class SamplesController < Projects::ApplicationController
 
     def index
       authorize! @project, to: :sample_listing?
+
+      @q = load_samples(@project.id).ransack(params[:q])
```

Comment:
We don't need to pass in the `@project.id` as a parameter to `load_samples`. Within `load_samples` you can just use @project.id in the where clause. `@project` is set in `Projects::ApplicationController` before any action so we can use it without having to pass it around

Reply:
Thanks, fixed in da02321a9ff3f1283cedc2f9f4706895b9c9d0fc